src/sql/workbench/contrib/query/browser/keyboardQueryActions.ts

```diff
@@ -208,7 +208,7 @@ export class CopyQueryWithResultsKeyboardAction extends Action {
 
 export class EstimatedExecutionPlanKeyboardAction extends Action {
 	public static ID = 'estimatedExecutionPlanKeyboardAction';
-	public static LABEL = nls.localize('estimatedExecutionPlanKeyboardAction', "Display Estimated Execution Plan");
+	public static LABEL = nls.localize('estimatedExecutionPlanKeyboardAction', "Execution Plan: Display Estimated Execution Plan");
```

Comment:
I believe you should be using the category parameter of registerWorkbenchAction, not embedding it (`Execution Plan`) in the label directly.